kit/logger/logger.go

```
@@ -0,0 +1,294 @@
+	case WarnLevel:
+		return 4
+	case ErrorLevel:
+		return 5
+	case FatalLevel:
+		return 6
+	}
+	return 0
+}
+
+// NewLayottoLogger creates new Logger instance.
+func NewLayottoLogger(name string) *layottoLogger {
```

golangci-lint

[warning] unexported-return: exported func NewLayottoLogger returns unexported type *logger.layottoLogger, which can be annoying to use

(revive)